### PR TITLE
Specs UI

### LINKED ISSUE #54: Add UI page-UI Module-111701015-111701006

### LINKED ISSUE #59: Add start session ui design page -111701016

### PR DESCRIPTION
**Pull request by sai-deekshith16**
Closes convict-git/KoraKaagaz#54
Closes convict-git/KoraKaagaz#59

### CODE REVIEW

**Comment by convict-git** [comment]
Guys any update about the status?

File: Specs/UI/SajithKumar-spec.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+public interface IDrawShapes {
+    // parameter: Pixel start - position of mouse drag started with RGB values
+    //                 Pixel end -  position of mouse drag ended with RGB values
+    void draw_circle(Pixel start, Pixel end);
+    void draw_square(Pixel start, Pixel end);         
+    void draw_rectangle(Pixel start, Pixel end); 
+    void draw_line(Pixel start, Pixel end);        
+    void draw_oval(Pixel start, Pixel end);    
+    void getSnapshot(Pixel start, Pixel end);
+}
```
Comment (convict-git) [inline Specs/UI/SajithKumar-spec.md]: Please change the comment styles and method names according to the coding guidelines.

**Review by convict-git — CHANGES_REQUESTED** [review]

File: Specs/UI/SajithKumar-spec.md (hunk 1 above)
Comment (sai-deekshith16) [reply Specs/UI/SajithKumar-spec.md]: I will send a new PR for that Spec. I will follow the provided guidelines from next time. Thank you.

**Comment by muhammedyaseen-git** [comment]
shall we close this as new PR has come @convict-git ?

**Comment by convict-git** [comment]
@muhammedyaseen-git I really don't understand what's going on!! They should commit to this PR itself.

File: Specs/UI/team-lead-spec.md [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+    //            float length - length of the sqaure
+    void drawSquare(Pixel start, float length);         
+    // parameter: Pixel start - top left position with RGB values for rectangle in whiteBoard
+    //            Pixel end - bottom right position with RGB values for rectangle in whiteBoard
+    void drawRectangle(Pixel start, Pixel end); 
+    // makes an object for a line segment parallel to the whiteBoard 
+    // parameter: Pixel start - start position with RGB values
+    //            Pixel end - end position with RGB values
+    void drawLine(Pixel start, Pixel end);        
+}
```
Comment (convict-git) [inline Specs/UI/team-lead-spec.md:185]: Please look at the interface [https://github.com/convict-git/KoraKaagaz/tree/master/Specs/Processing](https://github.com/convict-git/KoraKaagaz/tree/master/Specs/Processing) and make the changes.

**Review by convict-git — CHANGES_REQUESTED** [review]

File: Specs/UI/team-lead-spec.md (hunk 2 above)
Comment (sai-deekshith16) [reply Specs/UI/team-lead-spec.md:185]: I updated the Spec. Can you review it again.

**Comment by convict-git** [comment]
I looked at the view files along with @sai-deekshith16. It looks good now. Later on, we can add CSS to make it look more elegant. 
Merging for now!

**Review by convict-git — APPROVED** [review]

**Pull request merged**